LICENSE.txt

```diff
@@ -1,13 +0,0 @@
-BSD 3-clause license
-Copyright (c) 2015-2020, conda-forge contributors
```

Comment:
This is the license of the feedstock, which should be kept. This is different form the license of the package.